lib/cookie_policy_consent/blocs/cookie_policy_consent_bloc.dart

```diff
@@ -12,22 +14,34 @@ class CookiePolicyConsentBloc
   CookiePolicyConsentBloc(
     this.cookiePolicyConsent,
   ) : super(CookiePolicyConsentInitial()) {
-    on<CookiePolicyConsentEvent>((event, emit) async {
-      if (event is VerifyCookiePolicyConsent) {
-        emit(VerifyingCookieConsent());
+    on<CookiePolicyConsentEvent>(
+      (event, emit) async {
+        if (event is VerifyCookiePolicyConsent) {
+          emit(VerifyingCookieConsent());
 
-        final hasAcceptedCookiePolicyConsent =
-            await cookiePolicyConsent.hasAcceptedCookiePolicy();
+          final hasAcceptedCookiePolicyConsent =
+              await cookiePolicyConsent.hasAcceptedCookiePolicy();
 
-        if (hasAcceptedCookiePolicyConsent) {
+          if (hasAcceptedCookiePolicyConsent) {
+            emit(CookiePolicyConsentAccepted());
+          } else {
+            emit(CookiePolicyConsentRejected());
+          }
+        } else if (event is AcceptCookiePolicyConsent) {
+          cookiePolicyConsent.acceptCookiePolicy();
```

Comment:
If we await then we're sure it's OK when the local storage is full (or disabled) because it will throw so we won't emit Accepted
```suggestion
          await cookiePolicyConsent.acceptCookiePolicy();
```